src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/actions/RadiclePullAction.java

```diff
@@ -30,10 +30,12 @@ public void performAction(Project project) {
         updateCountDown = new CountDownLatch(repos.size());
         ApplicationManager.getApplication().executeOnPooledThread(() -> {
             if (BasicAction.isValidConfiguration(project)) {
-                var pull = new RadPull();
-                repos.forEach(repo -> ApplicationManager.getApplication().executeOnPooledThread(() ->
-                        new BasicAction(pull, repo, project, updateCountDown).perform()));
-                UpdateBackgroundTask ubt = new UpdateBackgroundTask(project, RadicleBundle.message(pull.getProgressBarTitle()),
+                repos.forEach(repo -> ApplicationManager.getApplication().executeOnPooledThread(()->{
```

Comment:
Here, I'm thinking if this action is correct. Where are we checking whether the repo is actually a rad-based one?